CHANGELOG.md

```diff
@@ -6,10 +6,11 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
-### Added
-- The `get_server_info` method is now available in SDK. It is only supported against Conjur enterprise server 
 ### Changed
-### Deprecated
-### Removed
-### Fixed
-### Security
+- Abstract authentication flow into new `AuthenticationStrategyInterface`
+  [conjur-api-python#20](https://github.com/cyberark/conjur-api-python/pull/20)
+
+## [8.0.0] - 2022-05-25
```

Comment:
Bit confused about this. 
1. How is it that this repo has never had any releases ?
2. 8.0.0 ?

Reply:
Basically there used to be one repo which included the python api and CLI, with the latest release being 7.1.0. Then the two were split into separate repos. So the python api repo has no releases, but is sort of starting out at 7.1.0. This PR introduces breaking changes to the API, so we're bumping it to 8.0. Meanwhile the CLI implementation of this won't have breaking changes so it'll be 7.2.

Reply:
Thanks!